src/Sylius/Bundle/ResourceBundle/test/src/AppBundle/Entity/Book.php

```diff
@@ -12,12 +12,18 @@
 namespace AppBundle\Entity;
 
 use Sylius\Component\Resource\Model\ResourceInterface;
+use Sylius\Component\Resource\Model\TranslatableInterface;
+use Sylius\Component\Resource\Model\TranslatableTrait;
 
 /**
  * @author Anna Walasek <anna.walasek@lakion.com>
  */
-class Book implements ResourceInterface
+class Book implements ResourceInterface, TranslatableInterface
 {
+    use TranslatableTrait {
+        __construct as private initializeTranslationsCollection;
```

Comment:
Aren't you forgetting to call `initializeTranslationsCollection` in the constructor?